src/Twig.php

```diff
@@ -104,7 +104,9 @@ public function fetch($template, $data = [])
      */
     public function render(ResponseInterface $response, $template, $data = [])
     {
-         return $response->getBody()->write($this->fetch($template, $data));
+         $response->getBody()->write($this->fetch($template, $data));
```

Comment:
Posted all in one issue: https://github.com/slimphp/Twig-View/issues/9

Btw i change the call because not need to get the body but write inside to response object directly.